W-20282179-rtf-2.11-remove-following-kt

kevintroller opened the pull request:
…tation

- Replaced ~250 instances of 'the following' with context-appropriate alternatives
- Used 'this' for singular references (e.g., 'this command', 'this guide')
- Used 'these' for plural references (e.g., 'these topics', 'these steps')
- Applied proper capitalization at sentence beginnings
- Modified 64 .adoc files across all RTF documentation sections

# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

tasmoore commented on modules/ROOT/pages/_partials/include-deploy-resource-allocation.adoc:
```diff
@@ -107,7 +107,7 @@ The startup time for a Mule application is correlated to the total number of vCP
 == Application Performance
 
-The resources allocated to your Mule application determine the application's performance. The following table lists approximate values for throughput based on the total number of vCPU cores allocated for a single Mule application performing simple processing on a 10-KB payload:
+The resources allocated to your Mule application determine the application's performance. These table lists approximate values for throughput based on the total number of vCPU cores allocated for a single Mule application performing simple processing on a 10-KB payload:
```
```suggestion
The resources allocated to your Mule application determine the application's performance. These tables list approximate values for throughput based on the total number of vCPU cores allocated for a single Mule application performing simple processing on a 10-KB payload:
```

tasmoore commented on modules/ROOT/pages/_partials/include-deploy-resource-allocation.adoc:
```diff
@@ -148,7 +148,7 @@ Performance test results are based on an Runtime Fabric cluster with controller
 A load generator based on C++, which has better efficiency in SSL connections was used to yield the maximum throughput.
 
-The following table summarizes the approximate requests (averaging 10 KB) that can be served with a single replica of the internal load balancer, based on the number of CPU cores. In most cases, Elliptical Curve Digital Signature Algorithm (ECDSA) provides double the performance of a 2K RSA key. Supported curves are secp521r1 (P-521), secp384r1 (P-384), and secp256r1, also known as prime256v1 (P- ...
+These table summarizes the approximate requests (averaging 10 KB) that can be served with a single replica of the internal load balancer, based on the number of CPU cores. In most cases, Elliptical Curve Digital Signature Algorithm (ECDSA) provides double the performance of a 2K RSA key. Supported curves are secp521r1 (P-521), secp384r1 (P-384), and secp256r1, also known as prime256v1 (P-256).
```
```suggestion
These tables summarize the approximate requests (averaging 10 KB) that can be served with a single replica of the internal load balancer, based on the number of CPU cores. In most cases, Elliptical Curve Digital Signature Algorithm (ECDSA) provides double the performance of a 2K RSA key. Supported curves are secp521r1 (P-521), secp384r1 (P-384), and secp256r1, also known as prime256v1 (P-256).
```

tasmoore commented on modules/ROOT/pages/_partials/include-deploy-resource-allocation.adoc:
```diff
@@ -205,7 +205,7 @@ ECDSA keys are also supported. In most cases, ECDSA doubles the performance of a
 
 CPU cores affect some thread pool settings and default max concurrency in Mule runtime engine.
-For example, Mule runtime engine 4.4 uses the following formula in `{MULE_HOME}/conf/scheduler-pools.conf` to calculate the thread pool size, which relies on the predefined cores parameters.
+For example, Mule runtime engine 4.4 uses these formula in `{MULE_HOME}/conf/scheduler-pools.conf` to calculate the thread pool size, which relies on the predefined cores parameters.
```
```suggestion
For example, Mule runtime engine 4.4 uses this formula in `{MULE_HOME}/conf/scheduler-pools.conf` to calculate the thread pool size, which relies on the predefined cores parameters.
```

tasmoore commented on modules/ROOT/pages/_partials/include-install-tasks.adoc:
```diff
@@ -7,25 +7,25 @@ Before you install the license key, encode it to Base64 format.
 === Encode the License Key
 
-* On MacOS, run the following command:
+* On MacOS, run these command:
```
```suggestion
* On MacOS, run this command:
```

tasmoore commented on modules/ROOT/pages/_partials/include-install-tasks.adoc:
```diff
@@ -7,25 +7,25 @@ Before you install the license key, encode it to Base64 format.
 ----
 +
-* On Unix, run the following command:
+* On Unix, run these command:
```
```suggestion
* On Unix, run this command:
```

tasmoore commented on modules/ROOT/pages/_partials/include-install-tasks.adoc:
```diff
@@ -7,25 +7,25 @@ Before you install the license key, encode it to Base64 format.
 ** Use a WSL or Cygwin shell that includes the base64 tool and use the above Unix command.
 ** Use the base64.exe program included with Windows git (C:\Program Files\Git\usr\bin).
-** Use the following Powershell command:
+** Use these Powershell command:
```
```suggestion
** Use this Powershell command:
```

tasmoore commented on modules/ROOT/pages/_partials/include-install-tasks.adoc:
```diff
@@ -34,7 +34,7 @@ $BASE64_ENCODED_LICENSE=[convert]::ToBase64String((Get-Content -path "license.li
 === Apply the License Key Using rtfctl
 
-. On the controller node acting as the leader during installation (the installer node), run the following command: 
+. On the controller node acting as the leader during installation (the installer node), run these command: 
```
```suggestion
. On the controller node acting as the leader during installation (the installer node), run this command: 
```

tasmoore commented on modules/ROOT/pages/_partials/include-install-tasks.adoc:
```diff
@@ -60,7 +60,7 @@ rtfctl get mule-license
 === Apply the License Key Using Helm
 
-To apply the license using Helm, run the following command:
+To apply the license using Helm, run these command:
```
```suggestion
To apply the license using Helm, run this command:
```

tasmoore commented on modules/ROOT/pages/_partials/include-manage-proxies.adoc:
```diff
@@ -32,7 +32,7 @@ The password must be URL encoded.
 * *<10.0.0.2>*: The IP address or hostname used to access the SOCKS5 proxy.
 * *<8080>*: The host port on the host where the SOCKS5 proxy is listening for requests.
-. To verify the change was successful, run the following command to output the current value of the monitoring proxy: 
+. To verify the change was successful, run these command to output the current value of the monitoring proxy: 
```
```suggestion
. To verify the change was successful, run this command to output the current value of the monitoring proxy: 
```

tasmoore commented on modules/ROOT/pages/_partials/include-manage-proxies.adoc:
```diff
@@ -55,7 +55,7 @@ You must restart the `rtf-agent` pod and any Mule app that requires a proxy. Alt
 
 . Log in to a node where `rtfctl` has been installed. 
-. Run the following command, replacing the placeholder values with the following:
+. Run these command, replacing the placeholder values with these:
```
```suggestion
. Run this command, replacing the placeholder values with these:
```

tasmoore commented on modules/ROOT/pages/_partials/include-manage-proxies.adoc:
```diff
@@ -72,7 +72,7 @@ The password must be URL encoded.
 * *<1.1.1.1:8888,2.2.2.2:9999>*: Specifies the `RTF_NO_PROXY` hosts and ports, delimited by commas.
 * (Optional) `--confirm`: Skips manual acknowledgement of the change. If not specified, the `rtfctl apply proxy` command prompts you to confirm the change before continuing.
-. To verify the change was successful, run the following command to output the current value of the HTTP proxy:
+. To verify the change was successful, run these command to output the current value of the HTTP proxy:
```
```suggestion
. To verify the change was successful, run this command to output the current value of the HTTP proxy:
```

tasmoore commented on modules/ROOT/pages/_partials/include-manage-proxies.adoc:
```diff
@@ -130,7 +130,7 @@ anypoint.platform.proxy_password=password
 cluster. You can also configure a proxy for each application. 
 
-You can set up a proxy on each connector (such as an HTTP requestor) or set up a proxy with application properties as shown in the following example. Setting up a proxy with application properties is less work if you have a large number of connectors in an application.
+You can set up a proxy on each connector (such as an HTTP requestor) or set up a proxy with application properties as shown in these example. Setting up a proxy with application properties is less work if you have a large number of connectors in an application.
```
```suggestion
You can set up a proxy on each connector (such as an HTTP requestor) or set up a proxy with application properties as shown in the example. Setting up a proxy with application properties is less work if you have a large number of connectors in an application.
```

tasmoore commented on modules/ROOT/pages/authorized-namespaces.adoc:
```diff
@@ -17,7 +17,7 @@ Runtime Fabric installs the Runtime Fabric agent (rtf-agent) in the Runtime Fabr
 === Example
 
-The following is an example of a Runtime Fabric agent ClusterRole that allows the agent to control all cron jobs across all namespaces in a cluster.
+These is an example of a Runtime Fabric agent ClusterRole that allows the agent to control all cron jobs across all namespaces in a cluster.
```
```suggestion
This is an example of a Runtime Fabric agent ClusterRole that allows the agent to control all cron jobs across all namespaces in a cluster.
```

tasmoore commented on modules/ROOT/pages/authorized-namespaces.adoc:
```diff
@@ -74,7 +74,7 @@ subjects:
 ----
 
-To use clustered apps in authorized namespaces mode, the following RoleBinding must be created:
+To use clustered apps in authorized namespaces mode, these RoleBinding must be created:
```
```suggestion
To use clustered apps in authorized namespaces mode, this RoleBinding must be created:
```

tasmoore commented on modules/ROOT/pages/authorized-namespaces.adoc:
```diff
@@ -102,6 +102,6 @@ You configure authorized namespaces when you install Runtime Fabric. Refer to th
 == Add New Namespaces
 
-If you update the authorized-namespaces ConfigMap with a new namespace, you must restart the Runtime Fabric agent deployment using the following command:
+If you update the authorized-namespaces ConfigMap with a new namespace, you must restart the Runtime Fabric agent deployment using these command:
```
```suggestion
If you update the authorized-namespaces ConfigMap with a new namespace, you must restart the Runtime Fabric agent deployment using this command:
```

tasmoore commented on modules/ROOT/pages/configure-horizontal-autoscaling.adoc:
```diff
@@ -98,7 +98,7 @@ Customizing topology spread with `whenUnsatisfiable: DoNotSchedule`, without the
 MuleSoft owns and applies the https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/[autoscaling^] policy for your Mule application deployments.
 
-The following example shows the CPU-based HPA policy used for Mule apps deployed on Runtime Fabric instances. Note that you can choose `minReplicas` and `maxReplicas` values from the Runtime Manager user interface. 
+These example shows the CPU-based HPA policy used for Mule apps deployed on Runtime Fabric instances. Note that you can choose `minReplicas` and `maxReplicas` values from the Runtime Manager user interface. 
```
```suggestion
This example shows the CPU-based HPA policy used for Mule apps deployed on Runtime Fabric instances. Note that you can choose `minReplicas` and `maxReplicas` values from the Runtime Manager user interface. 
```

tasmoore commented on modules/ROOT/pages/configure-horizontal-autoscaling.adoc:
```diff
@@ -223,14 +223,14 @@ You can also track when autoscaling events occurred through *Audit logs* in Acce
 image::rtf-hpa-rtm-2.png[Runtime Manager UI with Audit logs and Scaling status]
 
-The following is an example log payload:
+These is an example log payload:
```
```suggestion
This is an example log payload:
```

tasmoore commented on modules/ROOT/pages/configure-horizontal-autoscaling.adoc:
```diff
@@ -223,14 +223,14 @@ You can also track when autoscaling events occurred through *Audit logs* in Acce
 ----
 
-Additionally, you can track the scaled up replicas startup and the number of replicas of your Mule apps by running the following `kubectl` command in your terminal:
+Additionally, you can track the scaled up replicas startup and the number of replicas of your Mule apps by running these `kubectl` command in your terminal:
```
```suggestion
Additionally, you can track the scaled up replicas startup and the number of replicas of your Mule apps by running this `kubectl` command in your terminal:
```

tasmoore commented on modules/ROOT/pages/configure-horizontal-autoscaling.adoc:
```diff
@@ -239,7 +239,7 @@ kubectl get events | grep HorizontalPodAutoscaler
 Use the `kubectl get events` command in Kubernetes to retrieve events that occurred within the cluster. The command provides information about various activities and changes happening in the cluster, such as pod creations, deletions, and other important events.
 
-The `grep` command filters the output of `kubectl get events` for lines that contain the string `HorizontalPodAutoscaler`. The following example shows the command output that includes events related to a `HorizontalPodAutoscaler` with information about successful rescaling operations triggered by the HPA:
+The `grep` command filters the output of `kubectl get events` for lines that contain the string `HorizontalPodAutoscaler`. These example shows the command output that includes events related to a `HorizontalPodAutoscaler` with information about successful rescaling operations triggered by the HPA:
```
```suggestion
The `grep` command filters the output of `kubectl get events` for lines that contain the string `HorizontalPodAutoscaler`. This example shows the command output that includes events related to a `HorizontalPodAutoscaler` with information about successful rescaling operations triggered by the HPA:
```

tasmoore commented on modules/ROOT/pages/configure-local-registry.adoc:
```diff
@@ -15,7 +15,7 @@ You cannot use local registry for installation or management of Runtime Fabric o
 If you set up a local registry, you can upgrade to a version later than 1.12.0 without reinstalling Runtime Fabric.
 +
-To add local registry to an existing Runtime Fabric instance, ensure that the installation is Helm-based xref:migrate-helm.adoc[] and perform the steps on the following section to add local registry.
+To add local registry to an existing Runtime Fabric instance, ensure that the installation is Helm-based xref:migrate-helm.adoc[] and perform the steps on these section to add local registry.
```
```suggestion
To add local registry to an existing Runtime Fabric instance, ensure that the installation is Helm-based xref:migrate-helm.adoc[] and perform the steps in this section to add local registry.
```

tasmoore commented on modules/ROOT/pages/configure-local-registry.adoc:
```diff
@@ -25,7 +25,7 @@ To add local registry to an existing Runtime Fabric instance, ensure that the in
 
 
-* If you use a local registry that needs authentication, synchronization and propagation of a pull secret between different namespaces is your responsibility. Alternatively, Runtime Fabric can synchronize your pull secret across different namespaces if your secret uses the following label:
+* If you use a local registry that needs authentication, synchronization and propagation of a pull secret between different namespaces is your responsibility. Alternatively, Runtime Fabric can synchronize your pull secret across different namespaces if your secret uses these label:
```
```suggestion
* If you use a local registry that needs authentication, synchronization and propagation of a pull secret between different namespaces is your responsibility. Alternatively, Runtime Fabric can synchronize your pull secret across different namespaces if your secret uses this label:
```

tasmoore commented on modules/ROOT/pages/configure-local-registry.adoc:
```diff
@@ -57,7 +57,7 @@ Automated registry synchronization is available from Runtime Fabric version 2.6.
 Ensure that your xref:access-management::permissions-by-product.adoc#runtime-manager[Access Management permission for Runtime Manager] is set to *Read Runtime Fabric*, which enables you to query Runtime Fabrics in your organization.
 
-. In your Unix shell, run the following script, replacing `agent-version`, `authorization-token`, `mulesoft-docker-server`, and any other parameter value where appropriate:
+. In your Unix shell, run these script, replacing `agent-version`, `authorization-token`, `mulesoft-docker-server`, and any other parameter value where appropriate:
```
```suggestion
. In your Unix shell, run this script, replacing `agent-version`, `authorization-token`, `mulesoft-docker-server`, and any other parameter value where appropriate:
```

tasmoore commented on modules/ROOT/pages/configure-local-registry.adoc:
```diff
@@ -110,7 +110,7 @@ Synchronize Manually::
 To synchronize the dependencies images manually, follow these steps:
 
-. Log in to your private container registry and run the following command, replacing `docker-server` where appropriate:
+. Log in to your private container registry and run these command, replacing `docker-server` where appropriate:
```
```suggestion
. Log in to your private container registry and run this command, replacing `docker-server` where appropriate:
```

tasmoore commented on modules/ROOT/pages/configure-local-registry.adoc:
```diff
@@ -198,7 +198,7 @@ You should see a message that the login was successful.
 ----
 +
-Alternatively, if you use authentication to access your registry, use the following command:
+Alternatively, if you use authentication to access your registry, use these command:
```
```suggestion
Alternatively, if you use authentication to access your registry, use this command:
```

tasmoore commented on modules/ROOT/pages/configure-local-registry.adoc:
```diff
@@ -222,7 +222,7 @@ Helm::
 . In Runtime Manager, select the *Helm* path, and follow the instructions on the screen platform.
 . Obtain your private `docker-server`, `docker-username`, and `docker-password`.
-. Follow the *Helm* path instructions in Runtime Manager to create a secret with your docker server name, username, and password by running the following command in your Unix shell:
+. Follow the *Helm* path instructions in Runtime Manager to create a secret with your docker server name, username, and password by running these command in your Unix shell:
```
```suggestion
. Follow the *Helm* path instructions in Runtime Manager to create a secret with your docker server name, username, and password by running this command in your Unix shell:
```

kevintroller committed:
Replace 'the following' with 'this' or 'these' based on singular/plural agreement

- Replaced 'the following' with 'this' for singular nouns (e.g., 'this command', 'this example')
- Replaced 'the following' with 'these' for plural nouns (e.g., 'these steps', 'these options')
- Ensured proper capitalization at the beginning of sentences
- Processed 57 .adoc files with 191 total replacements
